eva-lib/src/main/java/uk/ac/ebi/eva/lib/configuration/SpringDataMongoDbProperties.java

```diff
@@ -0,0 +1,64 @@
+package uk.ac.ebi.eva.lib.configuration;
+
+import org.hibernate.validator.constraints.NotBlank;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@ConfigurationProperties(ignoreUnknownFields = false, prefix = "spring.data.mongodb")
+@Component
+public class SpringDataMongoDbProperties {
+
+    @NotBlank
+    private String host;
+
+    @NotBlank
```

Comment:
This can be blank in any infrastructure that doesn't have user information centralized in a single database, e.g. local deployments.